src/index.css

```diff
@@ -17,6 +17,25 @@
   --section-description: #757575;
 }
 
+/* GOOGLE FONTS START*/
+
+@import url('https://fonts.googleapis.com/css2?family=Khand:wght@500&family=Montserrat:wght@700&family=Source+Sans+3:wght@200;300;400;600;700;900&display=swap');
```

Comment:
Try to always import fonts in index.html file instead of importing them in CSS using @import statement. @import is considered slower than HTML import.